site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -161,88 +161,105 @@
 
             <div id="gradeables" class="customization_item">
                 <h2>Gradeables</h2>
-                <div id="gradeables_collapse" class="collapsible_area">
```

Comment:
Is this div needed?  There still appears to be a reference to the ID `gradeables_collapse` in `rainbow_customization.js`.

Reply:
No, as mentioned in original description, there will be a follow up PR just to remove all the gradeables_collapse. In this PR just one has been removed for functionality of buckets.